conda-forge.yml

```diff
@@ -3,3 +3,7 @@ build_platform:
 conda_forge_output_validation: true
 provider:
   win: azure
+remote_ci_setup:
+  - conda-forge-ci-setup=3
+  - conda-forge/label/lief_dev::py-lief
```

Comment:
nice

Reply:
This was a real pleasure to see working 😍